functions/src/import/config.ts

```diff
@@ -55,10 +55,10 @@ const validateInput = (
   regex: RegExp,
   sizeLimit: number
 ) => {
-  if (!value || value === '' || value.trim() === '') {
+  if (!value || value === '' || !value.trim()) {
     return `Please supply a ${name}`
   }
-  if (value.length >= sizeLimit) {
+  if (value.length > sizeLimit) {
```

Comment:
String does not modify the original string, [per documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim)

 > returns a new string, without modifying the original string

That means that the string you did on line 58 did not change the string `value`.  So when you check 
`value.length > sizelimit` you could still be checking 
```
"           a.        " 
```

```suggestion
  if (value.trim().length > sizeLimit) {
```